api/api/views/viewsets/mission.py

```diff
@@ -20,11 +23,42 @@ class NmapViewset(viewsets.ModelViewSet):
     serializer_class = NmapSerializer
     permission_classes = [permissions.IsAuthenticated, IsOwner, IsManager & ReadOnly | IsPentester]
 
+    @swagger_auto_schema(
+        operation_description="Creates and parses an NMAP output object.",
+        request_body=openapi.Schema(
+            type=openapi.TYPE_OBJECT,
+            required=['ips'],
+            properties={
+                'recon_id': openapi.Schema(type=openapi.TYPE_INTEGER,
+                                           description="Id of recon"),
+                'ips': openapi.Schema(type=openapi.TYPE_ARRAY,
```

Comment:
this is incorrect, if you look a the tests you will see that we actually expect an nmap output.